### PR TITLE
Adding a patch script to replace paths for mesos

### PR DESCRIPTION
**Pull request by ashwanthkumar**
Closes https://github.com/ind9/mesos-buildpack-python/pull/1

@manojlds This supersedes the changes from https://github.com/ind9/mesos-buildpack-python/pull/1.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -7,18 +7,22 @@ ENV USER_ID=999 \
...
 
 RUN apt-get update  \
     && apt-get -y install coreutils curl tar git-core daemontools gcc g++ python
+
 RUN curl --silent http://dl.gliderlabs.com/herokuish/latest/linux_x86_64.tgz \
     | tar -xzC /bin
-RUN /bin/herokuish buildpack install  # Install officially supported buildpacks
+
+RUN /bin/herokuish buildpack install  \
+    && curl -L https://raw.githubusercontent.com/ind9/python-stack/mesos-fix/mesos-fix > /bin/mesos-fix \
```
Comment (manojlds) [inline Dockerfile]: can't this be `COPY`ied into the container?

Comment (ashwanthkumar) [inline Dockerfile]: I initially had an `ADD` command but it wasn't working on the docker hub as expected :(

Comment (manojlds) [inline Dockerfile]: @ashwanthkumar even official images do this. Example - https://github.com/docker-library/postgres/tree/8e867c8ba0fc8fd347e43ae53ddeba8e67242a53/9.5

Shouldn't be a problem ideally. Try `COPY` anyway. It is not recommended to use `ADD`

**Comment by ashwanthkumar** [comment]
@manojlds Thanks for that. `COPY` worked like a charm. Updated with the feedback.

**Comment by manojlds** [comment]
LGTM 👍

**Pull request merged**